### PR TITLE
Do not require reading event to listen for NFC content

### PR DESCRIPTION
**Pull request by beaufortfrancois**
FIC https://github.com/w3c/web-nfc/issues/304

***
<a href="https://pr-preview.s3.amazonaws.com/beaufortfrancois/web-nfc/pull/305.html" title="Last updated on Aug 22, 2019, 9:36 AM UTC (0a83f3f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-nfc/305/9ded2e9...beaufortfrancois:0a83f3f.html" title="Last updated on Aug 22, 2019, 9:36 AM UTC (0a83f3f)">Diff</a>

### CODE REVIEW

**Review by kenchris — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -2956,10 +2956,10 @@ <h3><dfn>Writing or pushing content</dfn></h3>
 
   <section> <h3>Listening for content</h3>
     <p>
-      In order to receive <a>NFC content</a>, the client needs to attach an
-      event listener for the "`reading`" event on an
-      {{NFCReader}} instance and then activate it by calling
-      <a>NFCReader.scan()</a>.
+      To listen for <a>NFC content</a>, the client MUST activate an
+      {{NFCReader}} instance by calling <a>NFCReader.start()</a>. When attaching
+      an event listener for the "`reading`" event on it, <a>NFC content</a> is
```
Comment (zolkis) [inline index.html:2961]: start() will or has become scan()?

Comment (beaufortfrancois) [reply index.html:2961]: `scan`: https://w3c.github.io/web-nfc/#the-scan-method

Comment (zolkis) [reply index.html:2961]: Confused since this text uses `NFCReader.start()`, shouldn't that be `NFCReader.scan()`?

Comment (beaufortfrancois) [reply index.html:2961]: I was confused. Sorry for this.
I've sent https://github.com/w3c/web-nfc/pull/306

**Pull request merged**